src/modules/users/dto/userList.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsNumber } from 'class-validator';
+import { Type } from "class-transformer";
+
+export class UserListDto {
+  @Type(() => Number)
```

Comment:
Why do you need '@Type'

Reply:
In order to convert number from string format to number format